fala/apps/adviser/views.py

```diff
@@ -4,11 +4,36 @@
 from django.conf import settings
 from django.urls import resolve, reverse
 from django.views.generic import TemplateView, ListView
+from django.http import HttpResponse
+import os
 
 from .forms import AdviserSearchForm, AdviserRootForm
 from .laa_laa_paginator import LaaLaaPaginator
 from .regions import Region
 
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+
+def robots_txt(request):
```

Comment:
I was thinking about consistency as the imports in the URL side doesn't match or other URL's

Up to you, but we could put it into a class to keep the consistency. Happy to discuss your thought on this.
```
class RobotsTxtView(View):
    def get(self, request):
        environment = os.getenv("ENVIRONMENT", "development").lower()

        if environment == "production":
            content = "User-agent: *\nDisallow:\n"  # Allow all
        else:
            content = "User-agent: *\nDisallow: /\n"  # Disallow all for staging/UAT

        return HttpResponse(content, content_type="text/plain")
```